bpf/lib/conntrack.h

```diff
@@ -718,27 +718,24 @@ static inline int __inline__ ct_create4(void *map, struct ipv4_ct_tuple *tuple,
 	if (map_update_elem(map, tuple, &entry, 0) < 0)
 		return DROP_CT_CREATE_FAILED;
 
-	if (ct_state->addr) {
+	if (ct_state->addr && ct_state->loopback) {
```

Comment:
This seems good to me.

I have just one thought, do we still need to create two entries for the loopback case?

Reply:
We need two entries in loopback case. Non-loopback case had used it in a completely different intention which this patch removes. OTOH we might have some room to refactor to make things simple around this `ct_state` use.